### ISSUE TITLE
Setting to disable splash screen

### ISSUE THREAD

**Issue by talosgl** [writemonkey/wm3#420]
Sorry if I am missing this. Is there--or can there be-- a setting to disable the splash screen at launch? I found "start_in_full_screen" which almost gets me at parity behavior for my WM 2 experience but would also like to disable splash so it launches right into the blank scratch doc if possible. Thanks!

**Comment by writemonkey** [comment]
No, not possible. The splash screen is there for a reason as ui is setting up behind it after the app starts up. After the ui loads you can also dismiss the splash screen by pressing any key.
i.

**Comment by talosgl** [comment]
I see, thank you for the answer!
